lib/helpers/ConstantHelper.ts

```diff
@@ -6,7 +6,7 @@
     settings: "Settings",
     packages: "Packages",
     members: "Members",
-    dictionary: "Dictionary",
+    dictionary: "Translation",
```

Comment:
```suggestion
    translation: "Translation",
```

Reply:
I would suggest keeping "dictionary" for now to avoid updating the acceptance test code. I will create another PR later to change all occurrences of "dictionary" to "translation," including the UI/API helper files and the acceptance tests.

Reply:
Sounds good! 💪